browser/ui/brave_shields_data_controller.cc

```diff
@@ -156,7 +164,25 @@ void BraveShieldsDataController::SetBraveShieldsEnabled(bool is_enabled) {
 }
 
 GURL BraveShieldsDataController::GetCurrentSiteURL() {
-  return web_contents()->GetLastCommittedURL();
+  GURL url = web_contents()->GetLastCommittedURL();
+
+#if BUILDFLAG(ENABLE_IPFS)
+  auto* ctx = web_contents()->GetBrowserContext();
+
+  if (ipfs::IpfsServiceFactory::IsIpfsEnabled(ctx) &&
```

Comment:
we don't want these checks spread throughout the code, the value in the urlbar is the VirtualURL and that's probably what we should be using for these checks

Reply:
if we need any ipfs specific code in BraveShieldsDataController then there's a problem we need to address in ipfs because it's too fragile to have these checks all over the place

Reply:
removed ipfs related code as it will need to be addressed in a separate issue